src/main/java/uk/co/ramp/policy/isolation/BoundedDistributionDeserializer.java

```diff
@@ -27,7 +26,7 @@ public BoundedDistributionDeserializer(RandomGenerator rng) {
 
   @Override
   public BoundedDistribution deserialize(JsonParser jsonParser, DeserializationContext ctxt)
-      throws IOException, JsonProcessingException {
```

Comment:
Would the isolationPropertiesReader throw the JsonProcessingException? Or does the IOException catch the same error?

Reply:
It appears to be wrapped up as an IOException